pkgs/development/python-modules/pywebview/default.nix

```
@@ -17,18 +19,18 @@
 buildPythonPackage rec {
   pname = "pywebview";
   version = "4.3.3";
-  format = "setuptools";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
We have a new parameter for pyproject.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters